src/components/Tombstone.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <dl id="tombstone">
+    <!-- Registration Date -->
+    <template v-if="!!registrationDate">
+      <dt class="mr-2">Registration Date:</dt>
+      <dd id="registration-date">{{ registrationDate }}</dd>
+    </template>
```

Comment:
What do you think about creating a component for the items?  It looks like you have only two variants, a display version and an edit version.   The benefit is that if the business needs to modify the key fact displayed all we need to do is change the data supplied.  We don't have to add an if-then.

Reply:
There is no edit version here; only items that may or may not apply to the currently loaded entity.

Sorry if I don't understand your suggestion. What do you mean?